lib/phlex/component.rb

```diff
@@ -49,7 +67,14 @@ def #{tag_name}(*args, **kwargs, &)
       end
 
       def inherited(child)
+        if child.instance_method(:initialize).parameters.to_h[:req]
```

Comment:
Sadly this doesn't work because the child hasn't defined its own `initialise` method by this point.